CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.16.0 - 2020-12-04
+### Added
+- MongoDB server defaults to version “3.6”
```

Comment:
I would say something like `Newly created MongoDB servers default to version "3.6"`. 

Without the "newly created" part I could see people being confused about new vs existing servers and version 3.6

Reply:
Also mention Azure Cosmos DB in there somehow, as we have a lot of non-Azure users for Mongo and this will not affect them